specification/sql/resource-manager/Microsoft.Sql/preview/2015-05-01-preview/hybridLinkCreate.json

```diff
@@ -0,0 +1,280 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "2015-05-01-preview",
+    "title": "SqlManagementClient",
+    "description": "The Azure SQL Database management API provides a RESTful set of web APIs that interact with Azure SQL Database services to manage your databases. The API enables users to create, retrieve, update, and delete databases, servers, and other entities."
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "paths": {
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Sql/managedInstances/{managedInstanceName}/hybridLinkCreate": {
```

Comment:
creating something via a POST is not recommended in Azure. It cant be invoked via ARM template, Azure policy doesnt work against POST, its not idempotent etc. Can this be exposed via a PUT?

Reply:
and once you add the PUT, you will have to support GET, DELETE etc. which will enable the complete management experience for this. otherwise how does someone get this once it is created? 
another question, is this a singleton? can only one hybridLInk be created under a managedINstance?

Reply:
It can be exposed via PUT I guess.
We decided to implement delete support a bit later, as our feature will not support it in preview.
I'll implement a GET and it is not a singleton.

Btw thanks for the comments.